indexer/map_object.cpp

```diff
@@ -94,7 +94,23 @@ std::string MapObject::GetLocalizedType() const
   return platform::GetLocalizedTypeName(classif().GetReadableObjectName(copy.GetBestType()));
 }
 
-std::string MapObject::GetAllLocalizedTypes() const
+std::string MapObject::GetLocalizedAllTypes() const
+{
+  std::ostringstream oss;
+    
+  oss << GetLocalizedType();
+ 
+  std::string secondaryTypes = GetLocalizedSecondaryTypes();
+  if (!secondaryTypes.empty())
+  {
+      oss << feature::kFieldsSeparator;
+      oss << secondaryTypes;
+  }
+
+  return oss.str();
+}
+
+std::string MapObject::GetLocalizedSecondaryTypes() const
```

Comment:
These methods look very similar. Does it make sense to have only one method with a parameter to return either only the first or all localized types?

Reply:
I've combined `GetLocalizedAllTypes()` and `GetLocalizedSecondaryTypes()` into a single `GetLocalizedAllTypes(bool withMainType)`